tests/libvmi/factory.go

```diff
@@ -44,7 +44,7 @@ func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 		WithResourceMemory("512M"),
 		WithRng(),
 		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskFedora)),
-		WithCloudInitNoCloudUserData(configurePassword, true),
+		WithCloudInitNoCloudUserData(configurePassword, false),
```

Comment:
Why do we do this ? Your commit msg is not clear on that.

Is it because you want to be able to 'see' the contents of the VMIs ?